### ISSUE TITLE
[2.5] - Shortcuts to open options and actions menu are not working

### ISSUE THREAD

**Issue by antonbsa** [bigbluebutton/bigbluebutton#14744]
**Describe the bug**

Shortcuts to open options (**ALT + O**) and Open Actions menu (**ALT + A**) are not working

**Comment by antonbsa** [comment]
The same shortcuts are working in Firefox (just need to press the **SHIFT** key on both). Also, other developers report to me that the same shortcuts are working for them with Chrome.

**My Desktop:**
 - OS: Ubuntu 20.04.4 LTS 64-bit;
 - Chrome 99.0.4844.74;
 - Firefox 99.0;

**Comment by JoVictorNunes** [comment]
Hi @antonbsa! Is it still reproducible for you?

**Comment by antonbsa** [comment]
Hi @JoVictorNunes, thanks for testing it! I was trying to understand why I'm getting a different behavior and I think I found it. I recently installed the [Blackbox chrome extension](https://chrome.google.com/webstore/detail/blackbox-select-copy-past/mcgbeeipkmelnpldkobichboakdfaeon), which seems to use those shortcuts by default. If I disable the extension, I'm able to use the shortcuts with BBB again
